etc/adminhtml/system.xml

```diff
@@ -28,14 +28,31 @@
                     <config_path>clean_admin_menu/settings/rule_modules</config_path>
                     <comment>We will change the behavior of the allowed module list.</comment>
                 </field>
-                <field id="allowed_tabs" type="multiselect" translate="label comment" showInDefault="1" showInWebsite="0" showInStore="0" sortOrder="30" canRestore="1">
+                <field id="enable_marketplace" type="select" translate="label comment" showInDefault="1" showInWebsite="0" showInStore="0" sortOrder="30" canRestore="1">
```

Comment:
Also, what about moving these two new settings to a separate group named "Marketplace Menu Item" outside "General" and then rename

"Magento Marketplace menu item" to "Status"
and
"Move Magento Marketplace menu item" to "Move"

Reply:
Yes it does make sense, thank you :)